src/app/file/pixly_format.cpp

```diff
@@ -244,6 +119,20 @@ bool PixlyFormat::onLoad(FileOp* fop)
       sprite->folder()->addLayer(new LayerImage(sprite));
     }
 
+    // load image sheet
+    Document* sheet_doc = load_document(nullptr, base::replace_extension(fop->filename(),"png").c_str());
+    fop->setProgress(0.5);
+
+    Image* sheet = check(check(check(check(check(sheet_doc)->sprite())->layer(0))->cel(0))->image()); // do I need to check it all?
```

Comment:
No need to check this after a successful `load_document` with a `png` file (and lets hope it always behaves in the same way 😅 ).